challenges/prototypes.js

```diff
@@ -5,29 +5,44 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
+function CuboidMaker(attr) {
+  this.length = attr.length;
+  this.width = attr.width;
+  this.height = attr.height;  
+};
 
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
+CuboidMaker.prototype.volume = function(){
+  return (this.length * this.width * this.height);
```

Comment:
Don't forget to interpolate values for length, width, and height!

Ex:
return (2 * (`${this.length}` * `${this.width}` + `${this.length}` * `${this.height}` + `${this.width}` * `${this.height}`));